sys/shell/commands/sc_netif.c

```diff
@@ -733,6 +798,75 @@ int _netif_send(int argc, char **argv)
     return 0;
 }
 
+
+#ifdef MODULE_BLE_LL
+/* BLE shell commands */
+void _netif_send_ble(int argc, char **argv)
+{
+    kernel_pid_t dev;
+    PDU_header *pdu_hdr;
+    uint8_t pkt_type;
+    uint8_t payload_len = 0;
+    uint8_t payload[BLE_PAYLOAD_LEN_MAX];
+    ng_pktsnip_t *pkt = NULL;
+
+    /* parse arguments */
+    if(argc < 3) {
+       printf("usage: %s <if> <type>\n", argv[0]);
+       return;
+    }
+
+    /* parse interface */
+    dev = (kernel_pid_t)atoi(argv[1]);
+
+    if (!_is_iface(dev)) {
+        puts("error: invalid interface given");
+        return;
+    }
+
+    /* parse packet type */
+    pkt_type = atoi(argv[2]);
+
+    if (pkt_type == CONNECT_REQ_TYPE) {
+        puts("error: CONNECT_REQ type is not supported yet");
+        return;
+    }
+
+    /* put packet together */
+    if (argc > 3) {
+        bzero(&payload, BLE_PAYLOAD_LEN_MAX);
+        payload_len = ng_netif_addr_from_str(payload, sizeof(payload), argv[3]);
+
+        if (pkt_type == ADV_DIRECT_IND_TYPE || pkt_type == SCAN_REQ_TYPE) {
+            if (payload_len != BLE_ADDR_LEN) {
+                puts("error: invalid address argument length");
+                return;
+            }
+        }
+        pkt = ng_pktbuf_add(NULL, payload, payload_len, NG_NETTYPE_UNDEF);
+        pkt = ng_pktbuf_add(pkt, NULL, BLE_PDU_HDR_LEN, NG_NETTYPE_UNDEF);
+        } else {
```

Comment:
The indentation of the right curly bracket is wrong. Also please put `else` in a new line according to our coding conventions